WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardPresentPaymentsOnboardingViewModel.swift

```diff
@@ -33,7 +33,9 @@ final class CardPresentPaymentsOnboardingViewModel: ObservableObject, PaymentSet
                 self?.updateLearnMoreURL(state: result)
                 self?.reevaluateShouldShow(onboardingState: result)
             })
-            .handleEvents(receiveOutput: trackState(_:))
+            .handleEvents(receiveOutput: { [weak self] state in
+                self?.trackState(state)
+            })
```

Comment:
This is a great catch, it's not obvious at all that this captures self. I know it does, but it doesn't stand out.